locales/en/messages.json

```diff
@@ -2286,6 +2286,126 @@
     "auxiliaryModeLogicAND": {
       "message": "AND"
     },
+    "auxiliaryHelpMode_ARM": {
+        "message": "Use to toggle ARM.<br>Enables motors and flight stabilisation",
```

Comment:
I don't know if the `Use to toggle [MODE]` text is necessary. It is just near the name of the mode.  I think only the description is necessary.